### ISSUE TITLE
Optimize pdf file output

### ISSUE THREAD

**Issue by jgieseler** [jgieseler/solarmach#17]
While it is in principle already possible to save the figure as a (vectorized) pdf file, the resulting plot needs more fine-tuning. Especially, the fonts are too small and partly misaligned (in the legend). 

``` python
from solarmach import SolarMACH

body_list = ['STEREO-A', 'Earth', 'BepiColombo', 'PSP', 'Solar Orbiter', 'Mars']
vsw_list = [400, 400, 400, 400, 400, 400, 400]   # position-sensitive solar wind speed per body in body_list
date = '2021-10-28 15:15:00'
filename = 'Solar-MACH_'+date.replace(' ', '_')+'.pdf'

sm = SolarMACH(date, body_list, vsw_list)
fig, ax = sm.plot(numbered_markers=True, return_plot_object=True, outfile=filename)
```
PDF file:
[Solar-MACH_2021-10-28_15:15:00.pdf](https://github.com/jgieseler/solarmach/files/9647884/Solar-MACH_2021-10-28_15.15.00.pdf)

Corresponding PNG file:
![solarmach](https://user-images.githubusercontent.com/39489154/192326795-2e8e30af-0a97-420b-892d-f5ebc4888b68.png)

**Comment by johan12345** [comment]
Yeah, it may well be that `tight_layout()` and `bbox_inches="tight"` interfere with that. Or it could also be related to the `dpi=200` setting used while creating the figure...

Would be great if we can get the plots to look the same between PDF and PNG. Maybe it is also worth considering not to use `tight_layout()` and `bbox_inches="tight"` at all here to achieve reproducible figure size and margins.

**Comment by jgieseler** [comment]
Yes, I think the way to go is to set everything manually and forget the automatic things. As I mentioned, I think they don't do anything with pdfs, and then it's hard to have comparable results... At least the DPI settings are completely ignored for the vector graphics.
